cg/apps/lims/orderform.py

```diff
@@ -299,20 +287,26 @@ def parse_sample(raw_sample):
     return sample
 
 
-def relevant_rows(orderform_sheet):
+def relevant_rows(orderform_sheet: Worksheet) -> List[Dict[str, str]]:
```

Comment:
what is this row relevant for? Maybe there's a better name

Reply:
I'm all for rewriting and improving this part of the code. However in this PR I have focused on removing dependency on deprecated package xlrd, not to change the functionality